chroot/run.go

```diff
@@ -491,7 +491,13 @@ func runUsingChroot(spec *specs.Spec, bundlePath string, ctty *os.File, stdin io
 		return 1, err
 	}
 	defer func() {
-		undoIntermediates()
+		if undoErr := undoIntermediates(); undoErr != nil {
+			if err == nil {
```

Comment:
I don't think this check is necessary.
I think if you do
err = errors.Wrap(nil, undoErr.Error())
It will do the right thing.

Reply:
Unfortunately, it is necessary as `errors.Wrap(nil, ...)` always returns nil. See [docs](https://godoc.org/github.com/pkg/errors#Wrap).